packages/sanity/src/structure/comments/src/components/reactions/CommentReactionsUsersTooltip.tsx

```diff
@@ -32,10 +33,11 @@ interface UserDisplayNameProps {
 function UserDisplayName(props: UserDisplayNameProps) {
   const {currentUserId, isFirst, userId, separator} = props
   const [user] = useUser(userId)
+  const {t} = useTranslation(commentsLocaleNamespace)
 
   const isCurrentUser = currentUserId === userId
   const you = isFirst ? 'You' : 'you'
-  const content = isCurrentUser ? you : user?.displayName ?? 'Unknown user'
+  const content = isCurrentUser ? you : user?.displayName ?? t('comments.reaction-unknown-user')
```

Comment:
Could we add the translation to `you` here?

Reply:
Nice catch!